tests/test_base.py

```diff
@@ -52,6 +52,13 @@ def test_iconv_working(auto_container):
     )
 
 
+def test_group_nobody_working(auto_container):
```

Comment:
I am afraid we'll have to exclude this test or xfail it on the LTSS codestreams, as it's unlikely that this will get fixed there.

Reply:
added skip